.github/workflows/ci.yml

```diff
@@ -60,6 +60,29 @@ jobs:
       - name: Compile TypeScript
         run: yarn typescript
 
+  test-build-docs:
+    name: Build API docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup node and npm registry
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+          registry-url: 'https://registry.npmjs.org/'
+          cache: 'yarn'
+
+      - name: Install node_modules
+        run: yarn install --frozen-lockfile
+
+      - name: Install node_modules (example/)
+        run: yarn install --frozen-lockfile --cwd example
+
+      - name: Build docs
+        run: yarn docs
```

Comment:
But the out put is not used anywhere?

What's the intention? I think I am missing something.

Reply:
This command will fail if there is public API that is not documented.

Reply:
Ah, now I get you. I was interpreting this differently when I was reading it yesterday 😅 👍